src/irr/asset/CMeshManipulator.cpp

```diff
@@ -1534,6 +1534,19 @@ core::smart_refctd_ptr<ICPUBuffer> IMeshManipulator::idxBufferFromTrianglesFanTo
 	return nullptr;
 }
 
+void IMeshManipulator::loadNormalQuantCacheFromBuffer(E_QUANT_NORM_CACHE_TYPE type, asset::ICPUBuffer* buffer)
+{
+
+}
+void IMeshManipulator::addCacheContentsFromBuffer(E_QUANT_NORM_CACHE_TYPE type, asset::ICPUBuffer* buffer)
```

Comment:
actually take an `SBufferBinding` its because it would be handy to support an offset into the buffer from which to read.

Reply:
shouldn't it be range instead of single offset?